src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.__current_operation = None
 
         # If an error message is any of these strings, it ignores the length limit (STATUS_ERROR_MSG_SIZE_LIMIT_IN_CHARACTERS)
-        self.__ignore_error_message_restriction_compliance_strings = [Constants.TELEMETRY_AT_AGENT_NOT_COMPATIBLE_ERROR_MSG]
+        self.__ignore_error_message_restriction_compliance_strings = [Constants.TELEMETRY_NOT_COMPATIBLE_ERROR_MSG]
```

Comment:
Why ignore the message restrictions? Afaik, these restriction are what the agent has on a status file, so we should follow those for any of our file writes

Reply:
Duplicate comment